components/store/StoreHours.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View } from 'react-native';
+import Colors from '../../constants/Colors';
+import { constructHoursDict } from '../../lib/mapUtils';
+import { Body, TabSelected } from '../BaseComponents';
+
+/**
+ * @prop
+ * */
+
+export default function StoreHours({ hours }) {
+  // e.g. hours = "8:30am-8:30pm Mon-Sat"
+
+  const daysOfTheWeek = {
+    Sun: 'Sunday',
+    Mon: 'Monday',
+    Tu: 'Tuesday',
+    Wed: 'Wednesday',
+    Thurs: 'Thursday',
+    Fri: 'Friday',
+    Sat: 'Saturday',
+  };
+
+  const daysOfTheWeekFull = Object.values(daysOfTheWeek);
+
+  const today = new Date();
+  const todaysDayIndex = today.getDay();
+  const todaysDay = daysOfTheWeekFull[todaysDayIndex];
+
+  const hoursList = dictHours => {
+    return (
+      <View style={{ display: 'flex', flexDirection: 'row' }}>
+        {/* Days of the week */}
+        <View style={{ width: '50%' }}>
+          {daysOfTheWeekFull.map((day, _) => {
+            if (day === todaysDay) {
+              return <TabSelected>{day}</TabSelected>;
+            }
+            return <Body color={Colors.secondaryText}>{day}</Body>;
+          })}
+        </View>
+        <View style={{ width: '50%' }}>
+          {daysOfTheWeekFull.map((day, _) => {
+            if (day == todaysDay) {
```

Comment:
I think we should make sure all of these are `===` instead of `==` bc eslint haha